scripts/download-properties

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var path = require('path');
+var queue = require('queue-async');
+var request = require('request');
+var pump = require('pump');
+var features = require('geojson-stream').parse;
+var through = require('through2');
+var stringify = require('JSONStream').stringify;
+
+var inputDir = process.argv[2];
+var outputDirectory = process.argv[3];
+
+var metadata = require(path.resolve(inputDir, 'meta.json'));
+var files = require(path.resolve(inputDir, 'files.json'));
+
+getProperties(function (err) { if (err) { throw err; } });
+
+// Download every geojson, but save only the properties for each feature, i.e.
+// strip geometries
+function getProperties (callback) {
+  var q = queue(5);
+  var tasks = [];
```

Comment:
the build-list-of-tasks-and-then-defer-them technique isn't necessary, you should be able to defer in the loop itself.